mem.go

```diff
@@ -86,19 +87,13 @@ func (r RO) EqualBytes(b []byte) bool { return r.str() == string(b) }
 // Less reports whether r < r2.
 func (r RO) Less(r2 RO) bool { return r.str() < r2.str() }
 
-var builderPool = sync.Pool{
-	New: func() interface{} {
-		return new(strings.Builder)
-	},
+func (r RO) WriteTo(w io.Writer) (int, error) {
```

Comment:
Do not break compatibility with https://pkg.go.dev/io@go1.22.4#WriterTo

Reply:
But the problem is, `RO` is not a `io.Reader`, so you can't gain anything from impl `io.WriterTo`